docs/FAQ/gui-faq-sv.md

```diff
@@ -3,146 +3,199 @@ Zonemaster
 
 <a name="q2"></a>
 #### 2. Vem står bakom Zonemaster?
-
 Zonemaster är ett samarbetsprojekt mellan [Internetstiftelsen](https://internetstiftelsen.se/)
 (registry för .se och .nu) och [AFNIC](https://www.afnic.fr/en/)
-(registry för .fr och andra TLD:er som .re, .pm, .tf, .wf, .yt samt .paris).
+(registry för .fr och andra TLD:er som .re, .pm, .tf, .wf, .yt och .paris).
 
 <a name="q3"></a>
-#### 3. Hur kan Zonemaster hjälpa mig?
+#### 3. Hur kan Zonemaster hjälpa till?
 
 Zonemaster är framtaget för två kategorier av användare:
 
-  - Användare med DNS-kunskap.
+  - Användare med kunskap om DNS-standarden.
   - Användare som bara vill veta om domäner som hen äger eller använder
     har några problem eller inte.
 
 Användare av den andra kategorin bör vända sig till sin DNS-operatör
-så fort något inte lyser "grönt" vid en test.
+ifall det finns fel eller varningar för någon av testen av deras domännamn.
 
 <a name="q4"></a>
-#### 4. Zonemaster visar "Fel"/"Varning" när jag testar min domän, vad betyder det?
+#### 4. Zonemaster visar "Fel" eller "Varning" när jag testar min domän, vad betyder det?
 
-Det beror på vilket test det gäller.
+Det beror på vilken test det gäller. Varje testfall kommer med ett eller flera
+meddelanden som beskriver det problem som har funnits. Mera detaljer om varje
+testfall kan hittas via dokumentet "[Defined Test Cases]"
+(*Definierade testfall*).
 
 <a name="q5"></a>
 #### 5. Hur kan Zonemaster bedömma vad som är rätt och fel?
-
-Ingen kan ge ett definitivt, slutgiltigt utlåtande om en domäns hälsa. Detta är 
-viktigt att poängtera. Vi som står bakom Zonemaster påstår inte 
-att Zonemaster alltid har helt rätt. I vissa fall går åsikter isär, speciellt 
-mellan olika länder, men ibland även lokalt. Vi har tillsammans inom samarbetsprojektet
-gjort vårt allra bästa att ta fram en så bra som möjligt profil för hur dessa
-olika fel bedöms innan de presenteras för dig som användare av verktyget.
-
-En fördel för dig som användare är dock att det är enkelt att skapa en egen profil för hur
-allvarliga vissa fel ska vara, via CLI-verktyget går det att direkt peka ut sin egen profil.
-
-Men eftersom DNS utvecklas hela tiden kan situationer som idag bara kräver en 
-varning räknas som allvarliga fel imorgon. Om du tror du hittat något som du tycker
-vi felbedömt, tveka då inte att kontakta oss på
+Zonemasters bedömning är i första hand baserat på DNS-standarden som den är
+definierad i [RFC:er][RFCs] (text på engelska; det finns även en
+[beskrivning av RFC i engelskspråkiga Wikipedia][Wikipedia#Engelska#RFC]).
+Zonemaster baserar också sin bedömning på rekommenderade inställningar av DNS,
+vilka är mera löst definierade. Alla zonemastertest är definierade och finns som
+"[Test Case Specifications][Defined Test Cases]" (*testfallsspecifikationer*) i
+vilka det återfinns referenser till dokument med relevant del av DNS-standarden.
+En beskrivning av felnivåerna som används, som *notice* ("varsel"), *warning*
```

Comment:
Är verkligen "varsel" ett bra ord här? Det är väl mer en kommentar eller anmärkning?

Reply:
Updated to "notering". Is that OK?

Reply:
That's ok.